schema.graphql

```diff
@@ -23,11 +23,21 @@ enum Token {
     KBTC
 }
 
+type NativeToken {
+    token: Token!
+}
+
+type ForeignAsset {
+    asset: Int!
+}
```

Comment:
Btw are we able to extract the name / symbol / decimals from the `asset-registry` events? Is it worth having that in Squid?

Reply:
Probably can't hurt; but we didn't really store any token metadata in Squid and relied on the lib so far to map it to rich Monetary objects. In the UI the same approach will probably work for now, but you're right that Squid would be more usable (for general querying, outside out UI) if it could indeed supply the metadata itself. I think I'll add that to the backlog though.